Client/src/Pages/Maintenance/CreateNewMaintenance.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import Dropdown from "../../Components/Dropdown";
+
+const CreateNewMaintenance = () => {
+  const repeatComponents = {
+    "Don't repeat": [<div key={1}>a</div>, <div key={2}>b</div>],
+    "Repeat daily": [<div key={3}>c</div>, <div key={4}>d</div>],
+    "Repeat weekly": [<div key={4}>e</div>, <div key={6}>f</div>],
+  };
+
+  const repeatDropDown = [
+    { name: "Don't repeat" },
+    { name: "Repeat daily" },
+    { name: "Repeat weekly" },
+  ];
+
+  const [maintenanceOptions, setMaintenanceOptions] = useState("Don't repeat");
+
+  const [selectedOption, setSelectedOption] = useState(repeatDropDown[0]);
+
+  const handleChange = (e, newValue) => {
+    setMaintenanceOptions(newValue.name);
+    setSelectedOption(newValue);
+  };
+
+  return (
+    <div>
+      <div>Create a maintenance window</div>
+      <div>Your pings won&apos;t be sent in this time frame.</div>
+      <div>Repeat</div>
+      <div>
+        <Dropdown
```

Comment:
@terryjung0244 We discussed using the MUI `Select` component here, was there some difficulty with the implementation?

Reply:
I encountered an issue using the Dropdown MUI Select. I created a 'repeatDropDown' array with three objects to use as select options. When I tried to send the 'repeatDropDown' object in an array, I needed to keep the Dropdown component unchanged, particularly its value. Initially, the Dropdown's propTypes value was set to a string. However, I changed it to an object. There's no error so far, but I'm unsure if this change is allowed.

const repeatDropDown = [
    { name: "Don't repeat" },
    { name: "Repeat daily" },
    { name: "Repeat weekly" },
];

Dropdown.propTypes = {
  value: PropTypes.object.isRequired,
};

Reply:
What was the issue with the Select component?

We really don't want autocomplete here so it would be best if we could resolve the problem with Select.

It's a straightforward component so I'm sure we can resolve it.